.github/workflows/dependent-repos.yml

```diff
@@ -29,6 +29,8 @@ jobs:
     container: mobilecoin/builder-install:v0.0.32
     permissions:
       pull-requests: write
+    env:
+      SGX_MODE: SW
```

Comment:
that code path is gone now. It will use either a SIM/SW signing certificate or a known hardware signing certificate.
https://github.com/mobilecoinfoundation/mobilecoin/blob/50f101044cc4913bb3a155d715c1dc70543ff558/attest/verifier/src/lib.rs#L27

previously it would use DEV signing certificate when IAS_MODE was dev, https://github.com/mobilecoinfoundation/mobilecoin/blob/e09d548972933169f645fabd04bb0c9384cc772e/attest/verifier/src/lib.rs#L27